### PR TITLE
🐛 Make sure IconMenu can receive Box props

### PR DESCRIPTION
**Pull request by InstaK**
### Description

It was not possible to add margin on the IconMenu element, I assume this is because it's not a Box element. I'm not sure if this was added this way on purpose, if so, I'll read it in the comments I suppose 🤷‍♂️ otherwise, now I fixed it.

### Breaking changes

- None

### Changelog

Also: should I already add the changes in this changelog when creating the PR? @duivvv @lowiebenoot

### CODE REVIEW

**Comment by lowiebenoot** [comment]
Yes, you should add your changes to the `unreleased` section in the changelog file.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- Changed the element of `IconMenu` to our `Box` component ([@InstaK](<(https://github.com/Kemosabert)>) in [#324](https://github.com/teamleadercrm/ui/pull/324))
```
Comment (lowiebenoot) [inline CHANGELOG.md]: You forgot to change the link 😄

Comment (lowiebenoot) [reply CHANGELOG.md]: I also don't know why you added the extra `<( ... )>`

Comment (InstaK) [reply CHANGELOG.md]: what 😄 

 :ok: by @InstaK on 30/07/2018 14:45:48

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- Changed the element of `IconMenu` to our `Box` component ([@InstaK](https://github.com/InstaK) in [#324](https://github.com/teamleadercrm/ui/pull/324))
```
Comment (nickwaelkens) [inline CHANGELOG.md]: PP: a bit more clear wording:

- `IconMenu` now returns a `Box` component instead of a `div`

**Pull request merged**